client/src/includes/dialog.js

```diff
@@ -6,7 +6,7 @@ export const dialog = (
   dialogs.forEach((template) => {
     const html = document.documentElement;
     const templateContent = template.content.firstElementChild;
-    document.body.appendChild(templateContent);
+    template.insertAdjacentElement('afterend', templateContent);
```

Comment:
I initially implemented a way to specify a data attribute to select the dialog's root, but then I thought this is simpler and enough for most use cases. I don't know if it affects accessibility though (I assume not as it should already be handled by A11yDialog?)